layers/meta-balena-jetson/recipes-support/os-power-mode/os-power-mode/os-power-mode

```diff
@@ -96,6 +96,9 @@ else
         jetson-orin-nano-seeed-j3010)
             power_modes=(["low"]="2" ["mid"]="1" ["high"]="0")
             ;;
+        forecr-dsb-ornx-*)
```

Comment:
At least in L4T 36.3, the available power modes for Nano 4GB, 8GB and Orin NX 16 differ: https://docs.nvidia.com/jetson/archives/r36.3/DeveloperGuide/SD/PlatformPowerAndPerformance/JetsonOrinNanoSeriesJetsonOrinNxSeriesAndJetsonAgxOrinSeries.html#supported-modes-and-power-efficiency

For example, the Nano 4GB has 3 power modes whereas the 8GB only has 2, and the NX 16GB supports 4 modes

Reply:
Updated.